src/main/java/school/lemon/changerequest/java/introduction/hw2/Task5.java

```diff
@@ -2,10 +2,14 @@
 
 public class Task5 {
     public static boolean commondEnd(int[] a, int[] b) {
-        return false;
+
+        return a[0] == b[0] || a[a.length - 1] == b[b.length - 1];
```

Comment:
you haven't fully covered all possible cases. Please adjust your implementation regarding task description and given examples in Task5Specification